src/client/components/app-core/text.css

```diff
@@ -1,3 +1,20 @@
 html {
-  font-family: sans-serif;
+  font-family: 'system', sans-serif;
```

Comment:
Did you try this out on different devices? This is how it looks on my Android phone: 
![screenshot_20180620-171302](https://user-images.githubusercontent.com/1516059/41672935-8b326e82-74bb-11e8-8925-5b552ce640c8.png)